homeassistant/components/zwave_me/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow to configure ZWaveMe integration."""
+
+import logging
+
+from url_normalize import url_normalize
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import CONF_TOKEN, CONF_URL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ZWaveMeConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """ZWaveMe integration config flow."""
+
+    def __init__(self):
+        """Initialize flow."""
+        self.url = vol.UNDEFINED
+        self.token = vol.UNDEFINED
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        errors = {}
+        schema = vol.Schema(
+            {
+                vol.Required(CONF_URL): str,
+                vol.Required(CONF_TOKEN): str,
+            }
+        )
+
+        if user_input is not None:
+            if self.url == vol.UNDEFINED:
+                self.url = user_input["url"]
+            self.token = user_input["token"]
+            if not self.url.startswith(("ws://", "wss://")):
+                self.url = f"ws://{self.url}"
+            self.url = url_normalize(self.url, default_scheme="ws")
+            if "://" not in self.url:
+                errors[CONF_URL] = "invalid_url"
+                return self.async_show_form(
+                    step_id="user",
+                    data_schema=schema,
+                    errors=errors,
+                )
+
+            await self.async_set_unique_id(DOMAIN + self.url)
```

Comment:
The URL is normally not an ok unique id for the config entry.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-id-requirements